### PR TITLE
change OASIS manager info

### PR DESCRIPTION
**Pull request by deesto**
added myself as OASIS manager; added DN

### CODE REVIEW

File: virtual-organizations/BNL.yaml [hunk 1]
```diff
@@ -30,10 +30,9 @@ ID: 114
 LongName: Brookhaven National Laboratory
 OASIS:
   Managers:
-    Jason Federico:
+    John S. De Stefano Jr.:
       DNs:
-      - /DC=org/DC=opensciencegrid/O=Open Science Grid/OU=People/CN=Jason Federico 2576
-      - /DC=com/DC=DigiCert-Grid/O=Open Science Grid/OU=People/CN=Jason Federico 2576
+      -  /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=destefan/CN=665120/CN=John Steven De Stefano Jr
       ID: 9ad4d5aa3825356e2329e7fcb97c5b2fa2014b9e
```
Comment (DrDaveD) [inline virtual-organizations/BNL.yaml]: I'm not exactly sure how the ID is used, but judging from other files it appears your ID is 8a7d75ea65a2962520279197c72df6a16e1533a1 and it should be changed here.

Comment (brianhlin) [inline virtual-organizations/BNL.yaml]: @deesto if you accept this suggestion I think this PR looks pretty good

```suggestion
      ID: 8a7d75ea65a2962520279197c72df6a16e1533a1
```

Comment (brianhlin) [reply virtual-organizations/BNL.yaml]: Also, are we purposefully removing Jason Federico?

Comment (deesto) [reply virtual-organizations/BNL.yaml]: Yes, that's fine, thanks

File: virtual-organizations/BNL.yaml [hunk 2]
```diff
@@ -30,11 +30,10 @@ ID: 114
...
 OASIS:
   Managers:
-    Jason Federico:
+    John S. De Stefano Jr.:
       DNs:
-      - /DC=org/DC=opensciencegrid/O=Open Science Grid/OU=People/CN=Jason Federico 2576
-      - /DC=com/DC=DigiCert-Grid/O=Open Science Grid/OU=People/CN=Jason Federico 2576
-      ID: 9ad4d5aa3825356e2329e7fcb97c5b2fa2014b9e
+      -  /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=destefan/CN=665120/CN=John Steven De Stefano Jr
+       ID: 8a7d75ea65a2962520279197c72df6a16e1533a1
```
Comment (brianhlin) [inline virtual-organizations/BNL.yaml]: One whitespace fix:

```suggestion
      ID: 8a7d75ea65a2962520279197c72df6a16e1533a1
```

**Comment by deesto** [comment]
Thanks, all. I think it just needs to be manually merged now.

**Pull request merged**